location-service/src/test/java/uk/gov/cshr/locationservice/service/RegionLookupServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.cshr.locationservice.service;
+
+import uk.gov.cshr.locationservice.RegionLookup;
+import uk.gov.cshr.locationservice.UK_NUTS;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+public class RegionLookupServiceTest {
```

Comment:
agreed

Reply:
The tests check the correct Enum value is returned.  I can't see the merit in testing the actual name property of each enum value.  This might change anyway if we decide to return the Enum rather than the string which might be better for language translation.